index.js

```diff
@@ -143,15 +143,19 @@ module.exports = createStore =>
 			// FIXME: calculate this array at mount/unmount time
 			// iterate over mounted reducers, breadth-first
 			let paths = [];
-			mounts.forEach((mount, path) => paths.push(path.split('.')));
+			for (var path in mounts) {
+				if (mounts.hasOwnProperty(path)) {
```

Comment:
We control the definition of mounts and it's scoped entirely to this module, so I don't think we need this check.

Also, let's use `const` instead of `var`.